pages/api/users/update.ts

```diff
@@ -0,0 +1,56 @@
+import { PrismaClient } from "@prisma/client";
+import { NextApiRequest, NextApiResponse } from "next";
+import Joi from "joi";
+
+const prisma = new PrismaClient();
+type userDTO = {
+  id: number;
+  name: string;
+  email: string;
+  emailVerified: Date;
+  image: string;
+  hashedPassword: string;
```

Comment:
If this is an admin route, the `password` should be passed through the DTO here in plaintext and hashed within this endpoint. You can check out utils/hashPassword and https://github.com/calblueprint/ogsc/blob/main/pages/api/users/index.ts#L38 for example usage.

Reply:
also updated to return `sanitizeUser(user)` instead of just the user. Hope this is correct too!

Reply:
Yep, that's right! Thanks!